src/wallet/wallettool.cpp

```diff
@@ -54,7 +54,8 @@ static std::shared_ptr<CWallet> MakeWallet(const std::string& name, const fs::pa
     std::shared_ptr<CWallet> wallet_instance{new CWallet(nullptr /* chain */, name, std::move(database)), WalletToolReleaseWallet};
     DBErrors load_wallet_ret;
     try {
-        load_wallet_ret = wallet_instance->LoadWallet();
+        bool rescan_required = false;
+        load_wallet_ret = wallet_instance->LoadWallet(rescan_required);
```

Comment:
nit: Could leave it uninitialized to clarify this is an unused out param? (Same for other places)

```suggestion
        bool dummy;
        load_wallet_ret = wallet_instance->LoadWallet(/*rescan_required=*/dummy);
```

Reply:
Done :)